.github/workflows/sdks.yml

```diff
@@ -456,12 +471,16 @@ jobs:
           rm swift-nio/Tests/NIOFileSystemIntegrationTests/Test\ Data/*symlink
           cp -r swift-nio/Tests/NIOFileSystemIntegrationTests/Test\ Data/ swift-nio/Tests/NIOFileSystemIntegrationTests/FileHandleTests.swift pack
           cp yams/Tests/YamsTests/Fixtures/SourceKitten#289/debug.yaml pack
-          cp sdk-config/swift-*-android-x86_64-*${ANDROID_API_LEVEL}-sdk/usr/lib/lib*so pack/lib
-          cp sdk-config/swift-*-android-x86_64-*${ANDROID_API_LEVEL}-sdk/usr/lib/swift/android/lib*so pack/lib/swift/android
-          mv pack/lib/libc++_shared.so pack/lib/swift/android
+          if ${{ steps.cache-bundle.outputs.cache-hit != 'true' }}; then
+            cp sdk-config/swift-*-android-x86_64-*${ANDROID_API_LEVEL}-sdk/usr/lib/lib*so pack
+            cp sdk-config/swift-*-android-x86_64-*${ANDROID_API_LEVEL}-sdk/usr/lib/swift/android/lib*so pack
+          else
+            cp ~/.${{ startsWith(matrix.os, 'ubuntu') && 'config/' || '' }}swiftpm/swift-sdks/${{ steps.version.outputs.tag }}-android-${ANDROID_API_LEVEL}-*.artifactbundle/swift*sdk/android*sysroot/usr/lib/$(echo ${{ matrix.arch }} | sed "s/v7//")-linux-android*/${ANDROID_API_LEVEL}/lib*so pack
+            rm pack/lib{c,dl,log,m}.so
```

Comment:
@marcprux, these are stub libraries provided by the NDK just for linking, so I remove them so the Swift SDK libraries don't use them by mistake instead.

There is also a stub `libz.so` which collides with the Termux `libz.so` you recently added, so I put the latter in it's own `termux-zlib/` directory in the SDK bundle below.

Since that is only needed when running the `Foundation{Networking,XML}` libraries, which I don't test on the CI, I don't bother with `libz.so` here, but you will have to also throw out that stub and use the Termux one if testing these extra Foundation libraries.